Ported/dots-training-2022-03-na-group1_BucketBrigade/Assets/Scripts/Systems/FirePropagationSystem.cs

```diff
@@ -30,44 +30,100 @@ protected override void OnUpdate()
         
         float deltaTime = Time.DeltaTime * heatmapData.heatPropagationSpeed;
```

Comment:
Maybe calling this variable `deltaTime` is confusing.
I would rather name it `deltaHeat`.
What do you think?

Reply:
Yes. Will change.